### DISCUSSION TITLE
Spanish translation

### DISCUSSION THREAD

**Discussion by LuisLasbelin** [wildj79/foundryvtt-starfinder#797] [General]
*Spanish*

Estoy haciendo una traducción no oficial al español internacional. Necesito ayuda porque toma mucho tiempo hacerlo. 
Contesta por aquí con tu usuario de Discord o añádeme a LuisLasbelin#0193 si quieres colaborar.

**Comment by LebombJames** [comment]
It may be worth asking for help in the Foundry discord server https://discord.gg/foundryvtt
